runtime/bajun/src/lib.rs

```diff
@@ -956,6 +940,8 @@ construct_runtime!(
 		Migrations: pallet_migrations = 10,
 
 		// Monetary stuff.
+		Assets: pallet_assets::<Instance1> = 11,
+		AssetRegistry: pallet_asset_registry = 12,
```

Comment:
I would suggest moving these two indices to 18 and 19 so that we dont eat space for the System category above if we need it.

Reply:
Yeah, I wasn't 100% happy with my choice either. There is just one issue if we put it at 18/19. There is at least one more asset pallet coming if we want to connect to assethub, (which I suggest we do as we are already almost ready).

Should we move the asset related stuff to an entire new category, maybe?

Reply:
Hmm, yeah in that case it's probably better to do so